Core/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitor.cs

```diff
@@ -625,7 +630,7 @@ private bool IsCurrentExpressionEvaluatable (Expression expression)
     }
 #endif
 
-    private bool IsQueryableExpression (Expression expression)
+    private static bool IsQueryableExpression (Expression expression)
```

Comment:
To follow up from above :)

Historically speaking (and it's still in use today), we've had the rule to put the static members before all the instnace members. This always included the non-public stuff. So, finding the static members down in the instance section feels odd when you're trained to look for this sort of thing.

The maintainability: Yes, I did consider this on occasion, too, when it was especially important and there could be one or two places that are static and documented as static. But for the most part, I address this by simply making my types imutable. That way, it doesn't matter if a method accesses an instance field or not. Maybe I just got lucky, but the maintainability issue has never come up for me when I'm reasoning about the code. 

Funny thing, actually, seeing them as instance members actually makes them easier to put into context for me, given how I know they're just used by the instance members of my code.

Does that make any sort of sense to an outside set of eyes?